exercises/concept/blorkemon-cards/.docs/instructions.md

```diff
@@ -0,0 +1,80 @@
+# Instructions
+
+You suddenly feel a burst of nostalgia and a strong urge to dust off your old Plokemon™️ cards.
```

Comment:
I like the humorous tone, but I find the fact that the name is different every time a bit confusing. Maybe worth seeing if other people feel the same...

Reply:
Well, that's the whole joke, so if it is confusing, better tone it down.
I can ask for the opinion of our fellow maintainers :)